src/components/CurrentWalletAdvanced.tsx

```diff
@@ -68,45 +69,35 @@ export default function CurrentWalletAdvanced() {
   return (
     <div>
       {alert && <rb.Alert variant={alert.variant}>{alert.message}</rb.Alert>}
-      {isLoading && (
-        <rb.Row className="justify-content-center">
-          <rb.Col className="flex-grow-0">
-            <div className="d-flex mb-3">
-              <rb.Spinner as="span" animation="border" size="sm" role="status" aria-hidden="true" className="me-2" />
-              {t('current_wallet.text_loading')}
-            </div>
-          </rb.Col>
-        </rb.Row>
-      )}
+      {isLoading && <LoadingAccounts amount={5} />}
```

Comment:
Are you planning on reusing `LoadingAccounts` anywhere else? If not we could think about inlining it here similar to the placeholders in the other components.

Reply:
Changed